velox/functions/prestosql/aggregates/CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ velox_link_libraries(
   velox_functions_aggregates
   velox_functions_lib
   velox_functions_util
+  velox_functions_prestosql_impl
```

Comment:
Why do we need to link "velox_functions_prestosql_impl" to "velox_aggregates"?
Is it because of the json type? Then should not we link some type library instead or json type is in the "velox_functions_prestosql_impl"?

Reply:
Yes it's added for JSON. JsonType.h file is in the CPP library linked here

Reply:
Edit: Updated to use type library directly